taglib/asf/asffile.h

```diff
@@ -134,6 +134,7 @@ namespace TagLib {
       class ContentDescriptionObject;
       class ExtendedContentDescriptionObject;
       class HeaderExtensionObject;
+      class CodecListObject;
```

Comment:
So why do these forward declarations even need to be here? Kind of defeats the purpose of hiding everything under FilePrivate.